src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -6821,7 +6823,11 @@ public IContentWriter GetContentWriter(string path)
 
             try
             {
-                if (Directory.Exists(path))
+                if (Directory.Exists(path)
+#if !UNIX
+                    && string.IsNullOrEmpty(streamName)
```

Comment:
The same.

Reply:
On Windows (i.e., !UNIX), a directory can have data streams that are on it, not associated with any children of it. These data streams must be treated as content items, not child items.

How many places do you want the same comment? That feels very much like duplicating code.

Reply:
Comment added.  (Still feels like duplicating code, though.)